post_processing/compare_outputs.jl

```diff
@@ -63,12 +63,14 @@ compare(a::F, b::F, pn0 = "") where {F <: Fields.Field} =
     compare(true, a, b, pn0)
 function compare(zero_error, a::F, b::F, pn0 = "") where {F <: Fields.Field}
     isempty(propertynames(a)) || return _compare(zero_error, a, b, pn0)
-    err = abs.(Array(parent(a)) .- Array(parent(b)))
-    if !(maximum(err) == 0.0)
+    abs_err = abs.(Array(parent(a)) .- Array(parent(b)))
+    rel_err = abs.((Array(parent(a)) .- Array(parent(b))) ./ Array(parent(a)))
```

Comment:
I might suggest doing a norm in the denominator, otherwise any element that has zero will give us NaN

Reply:
What norm do you suggest? L-2?

Reply:
I think L2 is fine, we just need some ballpark that's likely not zero-- and if it is, we'll know we have a trivial (all zeros) field.

Reply:
Like 
```
rel_err = norm.( Array(parent(a)) .- Array(parent(b)) ) ./ norm.( Array(parent(a)) )
```
?

Reply:
Well, the maximum is computed below, and you don't want to broadcast norm (that would call norm over scalars), so perhaps:

```julia
rel_err = Array(parent(a)) .- Array(parent(b)) ./ norm( Array(parent(a)) )
```

Reply:
I would still keep the norm in the nominator. Otherwise we have a sort of mixed error definition (with the max below). By the way, the comparison jobs in CI are now not showing anything (even using a bitwise or). Can you think of why?

Reply:
> I would still keep the norm in the nominator. Otherwise we have a sort of mixed error definition
What's the error?

CI is not showing anything because we first check `if !(maximum(abs_err) == 0.0) | !(maximum(rel_err) == 0.0)`, so this must be returning false.

Reply:
> I would still keep the norm in the nominator. Otherwise we have a sort of mixed error definition (with the max below).

Computing the norm should result in a scalar variable, meaning that we wouldn't need a `maximum` call later.

Reply:
> CI is not showing anything because we first check `if !(maximum(abs_err) == 0.0) | !(maximum(rel_err) == 0.0)`, so this must be returning `false`?

That was not the reason why. I ran interactively in the REPL and saw the error message 
```
ERROR: UndefVarError: norm not defined
```

I am not sure why this was suppressed both in CI and when run locally (non interactively). For some reason the `stderr` was not redirected to `stdout`? I have never seen this behavior before. Took a while to figure out, because it only showed up in REPL mode. 

Now I fixed it. And it is printing the values correctly.